docker/README.md

```diff
@@ -123,3 +139,34 @@ server {
 }
 ```
 
+## Installing lizmap modules 
+
+### Install from archive
+
+- Mount `/www/lizmap/lizmap-modules/` at a location on the host or a from a named volume.
+- Get the archive of the lizmap module and extract it in the `/www/lizmap/lizmap-module/` or 
+  Add the module configuration snippet in the appropriate snippets directory (see the module documentation for  the configuration files to modify)
+- Add `<module_name>.access =2` to the `[modules]` section of `localconfig.ini.php` (which should be locatad in your `<install_dir>/lizmap/var/lizmap-config` directory
```

Comment:
> Add `<module_name>.access =2`

Well. no, it depends of the module and how it works. the access value could be set into `var/config/index/config.ini.php` and/or `var/config/admin/config.ini.php`  etc... So just say to follow the documentation of the module to set right values into the `[modules]` section of the right configuration file.

Reply:
Ok but that's weird that module activation configuration depends upon the module and is not unified...

Reply:
fixed

Reply:
its weird, this why I made improvements and I created this my-packages directory, I created a plugin for Composer jelix/composer-module-setup to support [this `modules-autoconfig-access-16` section in composer.json](https://github.com/3liz/lizmap-wps-web-client-module/blob/master/composer.json#L22) of modules ;-)